project/BuildHelper.scala

```diff
@@ -26,10 +26,10 @@ object BuildHelper {
   val Scala213: String   = versions("2.13")
   val ScalaDotty: String = "3.1.0" //versions.getOrElse("3.0", versions("3.1"))
 
-  val zioVersion        = "2.0.0-RC5"
-  val zioJsonVersion    = "0.3.0-RC7"
-  val zioPreludeVersion = "1.0.0-RC13"
-  val zioOpticsVersion  = "2.0.0-RC4"
+  val zioVersion        = "2.0.0-RC6"
+  val zioJsonVersion    = "0.3.0-RC8"
+  val zioPreludeVersion = "1.0.0-RC14"
+  val zioOpticsVersion  = "0.2.0-RC4"
```

Comment:
As an aside, the previous release version for zio-optics probably got messed up, it was `2.0.0-RC4`.